app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/model/DummyDatabaseManagement.kt

```diff
@@ -0,0 +1,145 @@
+package com.github.HumanLearning2021.HumanLearningApp.model
+
+import android.net.Uri
+import android.os.Parcelable
+import com.google.common.collect.ImmutableSet
+import kotlinx.parcelize.Parcelize
+import java.lang.IllegalArgumentException
+
+/**
+ * Dummy implementation of a database manager
+ * Dataset & category names and ids are equivalent
+ */
+data class DummyDatabaseManagement(val databaseService: DummyDatabaseService): DatabaseManagement {
+    companion object {
+        val staticDummyDatabaseManagement = DummyDatabaseManagement(DummyDatabaseService())
+    }
+
+    override suspend fun getPicture(category: Category): CategorizedPicture? {
+        return try {
+            databaseService.getPicture(category)
+        } catch (e: IllegalArgumentException) {
+            throw e
+        }
```

Comment:
Is this necessary?

Reply:
Yes because I decided not to deal with the exceptions in the dummy implementation

Reply:
but couldn't you remove the whole try/catch block? It would be equivalent no?